src/pages/Projects.js

```diff
@@ -1,18 +1,45 @@
 import React from 'react';
 import Header from '../components/Header';
-import ProjectTable from '../components/Project/ProjectTable';
-import ProjectAdd from '../components/Project/ProjectAdd';
+import ProjectsTable from '../components/projects/ProjectsTable';
+import AddProjectButton from '../components/projects/AddProjectButton';
 
-export default function Projects() {
-  return (
-    <React.Fragment>
-      <Header />
-      <div className='main-panel'>
-        <div className='content'>
-          <ProjectAdd />
-          <ProjectTable />
+export default class Projects extends React.Component {
+  static defaultProps = {
+    url: 'https://lesewert.herokuapp.com/api/v1',
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      projects: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch(this.props.url + '/projects')
```

Comment:
can be use 
`${this.props.url}/projects`

Reply:
fixed